src/set.ts

```diff
@@ -280,6 +300,15 @@ export class SuperchargedSet<T> implements Iterable<T> {
     return Array.from(this.set)
   }
 
+  /**
+   * Transforms this set into an array reversed.
+   *
+   * @returns {Array}
+   */
+  toReverseArray (): T[] {

```

Comment:
I prefer a `.reverse()` method on the set itself. The `.reverse()` method then returns a new set instance containing the items in reversed order.